website/docs/components/accordion/partials/code/component-api.md

```diff
@@ -0,0 +1,25 @@
+## Component API
+
+### Accordion
```

Comment:
[blocker] we're missing the description of the APIs of the `Accordion` (main component).

See suggestion below.